config/prod/jhu-biobank-nf-project.yaml

```diff
@@ -0,0 +1,24 @@
+stack_name: jhu-biobank-nf-project
+
+stack_tags:
+  Department: CompOnc
+  Project: Neurofibromatosis
+  OwnerEmail: bruno.grande@sagebase.org
+
+parameters:
+  S3ReadWriteAccessArns:
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/bruno.grande@sagebase.org
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/jineta.banerjee@sagebase.org
+  AccountAdminArns:
+    - {{stack_group_config.sso_admin_role.arn}}
+    - !stack_output_external workflows-nextflow-ci-service-account::ServiceRoleArn
+  TemplateRootUrl: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com'
+  TowerForgePolicyArn: !stack_output_external nextflow-forge-iam-policy::NextFlowForgePolicyArn
+  TowerLaunchPolicyArn: !stack_output_external nextflow-launch-iam-policy::NextFlowLaunchPolicyArn
+
+template_path: tower-project.yaml
+
+dependencies:
+  - prod/nextflow-vpc.yaml
+  - common/nextflow-forge-iam-policy.yaml
+  - common/nextflow-launch-iam-policy.yaml
```

Comment:
I have one major concern with these project stacks that I didn't get a chance to address yet. And it's not really these stacks but rather the accompanying script that creates the workspace and so on in Tower.

From what I've seen in the Tower database, we can create many things outside an organization. I think it might be best if we create the Sage-Bionetworks organization within Tower and ensure that everything is created inside it. @BrunoGrandePhD I will discuss this with you in chat.